privacyIDEAADFSProvider/AdapterPresentationForm.cs

```diff
@@ -26,6 +49,18 @@ public string GetFormHtml(int lcid)
             string otptext = "One-Time-Password";
             string submittext = "Submit";
             string htmlTemplate = Resources.AuthPage;
+
+            if(EnableEnrollment == true)
+            {
+                htmlTemplate = htmlTemplate.Replace("#ENROLLMENT#", EnrollmentText);
+                htmlTemplate = htmlTemplate.Replace("#ENROLLVAL#", EnrollmentUrl);
+                htmlTemplate = htmlTemplate.Replace("#ENROLLIMG#", EnrollmentImg);

```

Comment:
htmlTemplate = htmlTemplate.Replace("#ENROLLIMG#", "data:," + EnrollmentImg);

Reply:
I'm not sure I understand what this one does - isn't the `data` text already provided when pulling the data from the API? I originally had the img set to `data:,` because it was the equivalent of an empty image. In earlier revisions it was always shown, but now that it is hidden it makes sense for it to be empty upon creation.